common/eth2_network_config/built_in_network_configs/mainnet/config.yaml

```diff
@@ -50,6 +50,15 @@ CAPELLA_FORK_EPOCH: 194048  # April 12, 2023, 10:27:35pm UTC
 # Deneb
 DENEB_FORK_VERSION: 0x04000000
 DENEB_FORK_EPOCH: 269568  # March 13, 2024, 01:55:35pm UTC
+# Electra
+ELECTRA_FORK_VERSION: 0x05000000
+ELECTRA_FORK_EPOCH: 18446744073709551615
+# Sharding
+SHARDING_FORK_VERSION: 0x03000000
+SHARDING_FORK_EPOCH: 18446744073709551615
+
+# TBD, 2**32 is a placeholder. Merge transition approach is in active R&D.
+TRANSITION_TOTAL_DIFFICULTY: 4294967296
```

Comment:
Looks like these were accidentally added back. These configs were outdated and removed in #5233.

Reply:
Good catch thanks! [Should be fixed now](https://github.com/sigp/lighthouse/pull/5122/commits/ef724dc9e0a63bb574b3a1bf47c78c629b8a973b#diff-3c1b06d3c4c9fe3fc21011c3360db0d152b9516104377c9d6ebcc1484cc4fea3L56). I also added the Electra configs into the other networks.